protokt-codegen/src/main/kotlin/protokt/v1/codegen/generate/FileGenerator.kt

```diff
@@ -33,8 +34,19 @@ private class FileGenerator(
                 contents.info.kotlinPackage,
                 fileName(contents)
             ).apply {
-                addDeprecationSuppression()
-                // https://github.com/square/kotlinpoet/pull/533
+                addAnnotation(
+                    AnnotationSpec.builder(Suppress::class).apply {
+                        addMember("DEPRECATION".embed())
+                        // Suppresses a failure due to usage of @JvmStatic in common code. It seems prudent to add this
```

Comment:
This only actually triggered a problem in the multiplatform integration test project. So strange.